src/object_list.c

```diff
@@ -355,6 +355,11 @@ static int object_list_cache_load(int totalFiles, uint64 totalFileSize, int file
 	rct_plugin_header pluginHeader;
 	uint32 filterVersion = 0;
 
+#ifndef _WIN32
+	// TODO: remove me!
+	log_error("this is to be removed after testing/implementation is done");
```

Comment:
I remember running into some sort of problems when this function was enabled and allowed cache to be loaded, but I don't have a log, of what exactly the problem was, handy.
I could try checking that later on and perhaps it might even be fixed by now, but I wanted to make sure I make it easier to track all the hacks i've made to progress with linux port.